### PR TITLE
Feature load environment config

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Follow up PR (https://github.com/neo-project/neo/pull/851).
Use environment value of "NEO_NETWORK" to resolve which config.json should be loaded.

### CODE REVIEW

File: neo-cli/Extensions.cs [hunk 1]
```diff
@@ -3,7 +3,10 @@
 
 namespace Neo
 {
-    internal static class Helper
+    /// <summary>
+    /// extension methods
```
Comment (shargon) [inline neo-cli/Extensions.cs]:
```suggestion
    /// Extension methods
```

**Comment by Ashuaidehao** [comment]
> Is it still needed? I saw the conflicts and looks it was resolved.

Fixed confict.This feature could let user switch any private net easily,not just main or test net.

**Comment by shargon** [comment]
Hard to see your changes, was merged with mine changes

**Comment by shargon** [comment]
I can't sync for testing 
image

File: neo-cli/Settings.cs [hunk 2]
```diff
@@ -33,7 +33,7 @@ public static Settings Default
             {
                 if (_default == null)
                 {
-                    UpdateDefault(new ConfigurationBuilder().AddJsonFile("config.json").Build());
+                    UpdateDefault(Helper.LoadConfig("config"));
```
Comment (vncoelho) [inline neo-cli/Settings.cs]: config.json?

**Comment by Ashuaidehao** [comment]
> I can't sync for testing
> image

Sorry for making a mistake when push my branch, I'll close this PR and re-submit it later. 😭

File: neo-cli/Settings.cs (hunk 2 above)
Comment (Ashuaidehao) [reply neo-cli/Settings.cs]: https://github.com/neo-project/neo/blob/f927dec43409515e02f8156a6d22db9de3252601/neo/Helper.cs#L278-L285

Yes,the method will auto append ".json" file name.

**Pull request closed without merging**